5.2.piece_of_cake.py

```diff
@@ -0,0 +1,15 @@
+
+
+def get_recipe_price(prices, optionals=[], **ingredients):      # from 5.2
+    total = 0
+    for name, amount in ingredients.items():
+        if name in optionals:
+            continue
+        total += amount * (prices[name] / 100)

```

Comment:
You should aim to write everything in a single line, even if you can. 
This link https://towardsdatascience.com/you-should-stop-craving-for-python-one-liners-5dbf8e0de23d explains it well, even though I disagree with how important memory and runtime are in most python programs.